SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -48,6 +48,7 @@ struct CSAVariables {
   const su2double cb2_sigma = cb2 / sigma;
   const su2double cw1 = cb1 / k2 + (1 + cb2) / sigma;
   const su2double cr1 = 0.5;
+  const su2double CRot = 1.0; // It should be 2.0, but the HLPW5 wants 1.0
```

Comment:
We want consistency with the literature.
> For example, when using a value of 1 instead of 2, the (SA-noft2-R) would instead have the naming convention (SA-noft2-R(Crot=1)).
```suggestion
  const su2double CRot = 2.0;
```

Reply:
Another keyword CROT1 then? Or make CROT=2.0 a config option?